domain/src/main/java/org/oppia/domain/util/StateRetriever.kt

```diff
@@ -273,7 +274,7 @@ class StateRetriever @Inject constructor(
               .setNonNegativeInt(inputsJson.getInt(inputName))
               .build()
           )
-          else -> ruleSpecBuilder.putInput(inputName, createExactInputFromJson(inputsJson, inputName, interactionId))
+          else -> ruleSpecBuilder.putInput(inputName, createExactInputFromJson(inputsJson, inputName, interactionId,ruleSpecBuilder.ruleType))
```

Comment:
Nit: space after the ``,`` for consistency. Also, please make sure this fits within 100 characters (see my comment below for advice on wrapping lines of code to fit within 100 characters).

Reply:
done